domains/_github-pages-challenge-MrPop-rookie.snbhowmik.json

```diff
@@ -0,0 +1,18 @@
+{
+  "repo": "https://github.com/MrPop-rookie/MrPop-rookie.github.io",
+  "owner": {
+    "username": "MrPop-rookie",
+    "email": "snbhowmikac@gmail.com",
+    "linkedin": "https://linkedin.com/in/snbhowmik"
+  }
+  "record": {
+    "CNAME": "MrPop-rookie.github.io",
+    "TXT": "9a4944b6a34448f1edc9f4079db605"
```

Comment:
You only need TXT for Github Pages verification, go to https://www.is-a.dev/docs/github-pages/ and https://www.is-a.dev/docs/github-pages-verification/ to see what you need to do